File: JAVA/B201013/Lab1/palindrome.java

import java.util.*;
class check{
    int num,result=0;
    
    void palin(){
        int dup=num;
        while(dup!=0){
            result=(result*10)+dup%10;
            dup/=10;
        }
        if(result==num){
            System.out.println("Palindrome");
        }
        else{
            System.out.println("Not palindrome!");
        }
    }
    
}
public class palindrome {
    static int num;
    public static void main(String[] args){
        Scanner s=new Scanner(System.in);
        check Check=new check();
        System.out.print("Enter a number :");
        Check.num=s.nextInt();
        Check.palin();
        s.close();
    }
}

File: C/palindrome.c
//write a c program to check a number is palindrome are not
//ex 121,3663 are palindrome
#include<stdio.h>
int main(){
    int num,dn,sum=0,r;
    printf("Enter a Number :");
    scanf("%d",&num);
    dn=num;
    while(num!=0){
        r=num%10;
        sum=(sum*10)+r;
        num/=10;
    }
    if(dn==sum){
        printf("it's Palindrome");
    }
    else{
        printf("It's not Palindrome");
    }
    return(0);
}